connexion/cli.py

```diff
@@ -11,9 +12,25 @@
 
 logger = logging.getLogger('connexion.cli')
 CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
-
-
-def validate_wsgi_server_requirements(ctx, param, value):
+FLASK_APP = 'flask'
```

Comment:
Not sure if I understood purpose of these constants properly.
* We divide our config into **connexion apps**, **app frameworks** and **app servers** we serve our web-app with, right?
* Each app framework can be an app server at the same time, i.e serve itself
* We want to resolve dependencies between app servers and app frameworks (e.g flask can't be run with aiohttp by default although it has 3rd party [wsgi](https://github.com/etianen/aiohttp-wsgi) adapters)

Just trying to understand why we need so many configs which repeat the same names

Reply:
I don't know if I understand your points, but I will try to explain why I created these variables.

I created the AVAILABLE_SERVERS variable to ensure that the user input is right (but this validation will be done by the app when the App.run method is called). So this validation can be removed as well.

The AVAILABLE_APPS is for mapping the "--app-framework" input with the right class.

The DEFAULT_SERVERS was created because some frameworks has no default server (like the falcon framework, there's no way to run the application directly from falcon, a external wsgi server must be used), but for now the defaults are the same name of the framework, so I can remove this variable too if you want.